src/app/app.module.ts

```diff
@@ -63,18 +64,26 @@ export const angularticsSettings: Partial<Angulartics2Settings> = {
     Angulartics2Module.forRoot(angularticsSettings),
   ],
   providers: [
+    {
+      provide: APP_INITIALIZER,
+      useFactory: (cs: ConfigurationService) => () => cs.loadConfiguration(),
```

Comment:
I would suggest to use a different property name than cs because it resembles the CS locale. configServ at least or so...

Reply:
It is used only here, but whatever.. I will change it